lib/pure/collections/lists.nim

```diff
@@ -983,6 +983,16 @@ func toSinglyLinkedList*[T](elems: openArray[T]): SinglyLinkedList[T] {.since: (
   for elem in elems.items:
     result.add(elem)
 
+func toSinglyLinkedRing*[T](elems: openArray[T]): SinglyLinkedRing[T] {.since: (2, 0, 0).} =
```

Comment:
Why would this be `since: (2, 0, 0)`?

Reply:
I was mimicking the `toSinglyLinkedList` declaration.

I guess I don't fully understand when `since` should be used. Should I remove the `since`, or should the version be something else?